scripts/archive-artifacts.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
Maybe gather or prepare artefacts would be a better name as this script doesn't actually archive them, just prepares them in order for Jenkins to archive them.